engine/src/main/java/org/terasology/logic/inventory/ItemPickupAuthoritySystem.java

```diff
@@ -59,20 +66,32 @@ public void onDropItemEvent(DropItemEvent event, EntityRef itemEntity, ItemCompo
         if (!itemEntity.hasComponent(LocationComponent.class)) {
             itemEntity.addComponent(new LocationComponent(event.getPosition()));
         }
+
+        itemEntity.getComponent(ItemComponent.class).pickableUp = false;
+        long timeToPickUp = itemEntity.getComponent(ItemComponent.class).pickUpTime;
+        delayManager.addDelayedAction(itemEntity, DELAYED_PICKUP_ACTION_ID, timeToPickUp);
     }
 
+    @ReceiveEvent
+    public void onDelayedPickup(DelayedActionTriggeredEvent event, EntityRef entityRef) {
+        if (event.getActionId().equals(DELAYED_PICKUP_ACTION_ID)) {
+            entityRef.getComponent(ItemComponent.class).pickableUp = true;
```

Comment:
Is the saving of components relevant to the save system or the general flow of the game? I didn't know that component saving was necessary. Will do it from now on tho :D